x/tokenomics/keeper/token_logic_modules.go

```diff
@@ -194,6 +194,21 @@ func (k Keeper) ProcessTokenLogicModules(
 	}
 
 	// Execute all the token logic modules processors
+	// TODO_CRITICAL: Per-claim validator reward distribution causes significant precision loss.
```

Comment:
Is there a `TestPrecisionLossWithManyDistributions` test that we can add here that'll be resolved later?

Only add it if it's not too much work.